lib/cocoapods/installer/xcode/pods_project_generator.rb

```diff
@@ -203,16 +208,10 @@ def set_target_dependencies
               aggregate_target.native_target.add_dependency(pod_target.native_target)
               configure_app_extension_api_only_for_target(pod_target) if is_app_extension
 
-              pod_target.dependent_targets.each do |pod_dependency_target|
```

Comment:
method extracted. No logical changes. Helps support wiring up test dependencies to the native targets